.github/workflows/test.yaml

```diff
@@ -8,6 +8,9 @@ on:
       tag:
         required: false
         type: string
+      confdir:
```

Comment:
confirming this works and passes the value defined in the caller workflow to the workflow in the library.